pkg/apis/cluster/v1alpha1/cluster_types.go

```diff
@@ -62,6 +65,9 @@ type Connection struct {
 	// Will be populated by ks-apiserver if connection type is proxy.
 	KubernetesAPIEndpoint string `json:"kubernetesAPIEndpoint,omitempty"`
 
+	// External Kubernetes API Server endpoint
+	// Will be populated by ks-apiserver if connection type is proxy and ExternalKubeAPIEnabled is true.
```

Comment:
I believe this field is populated by tower, right?

Reply:
right, copied from the about comment, I'll change it